pkg/rke/k8s_rke_system_images.go

```diff
@@ -8418,6 +8418,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.21.1",
 		},
+		// Enabled out of band post v2.7.2
```

Comment:
Minor update to comment, same for the other introduced versions

```suggestion
		// Enabled with v2.7.5
```